bins/packages/qsfs/qsfs.sh

```diff
@@ -0,0 +1,30 @@
+QSFS_VERSION="0.2.0-rc1"
+QSFS_IMAGE="ghcr.io/threefoldtech/qsfs"
+
+download_qsfs() {
+    echo "download qsfs"
+    download_docker_image ${QSFS_IMAGE}:${QSFS_VERSION} qsfs-${QSFS_VERSION}.tar
+}
+
+prepare_qsfs() {
+    echo "[+] prepare qsfs"
+    github_name "qsfs-${QSFS_VERSION}"
+}
+
+install_qsfs() {
+    echo "[+] install qsfs"
+
+    mkdir -p "${ROOTDIR}"
+
+    extract_docker_image "${DISTDIR}/qsfs-${QSFS_VERSION}.tar" "${ROOTDIR}"
+}
+
+build_qsfs() {
+    pushd "${DISTDIR}"
```

Comment:
variable should be DESTDIR instead of DISTDIR

Reply:
why? this variable is set by the base build script in the project and is used everywhere (in all packages) so changing that here means i need to change it everywhere 
also i think distdir is from (distribution dir) so it is correct